src/components/copy/copy.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiCopy } from './copy';
+
+describe('EuiCopy', () => {
+  test('is rendered', () => {
+    const component = shallow(
+      <EuiCopy textToCopy="some text">
```

Comment:
The props says this component can also accept all of the common props. Can you add 
```suggestion
      <EuiCopy textToCopy="some text" {...requiredProps}>
```

and import `requiredProps` from `'../../test/required_props'`